src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/newlineatendoffile/Example4.cpp

```diff
@@ -1,12 +1,13 @@
 /*xml
 <module name="Checker">
   <module name="NewlineAtEndOfFile">
-    <property name="fileExtensions" value="java, xml, py"/>
+    <property name="fileExtensions" value="java, xml, cpp"/>
   </module>
 </module>
 */
 // violation 7 lines above 'File does not end with a newline.'
 // xdoc section -- start
-print("Hello world!")
-# no ⤶ below it is violation
+void main() { // ⤶
+    std::cout << "Hello World!";// ⤶
```

Comment:
This caused spelling violation.
Can we have some cpp code that is written in full English words?

Reply:
sure we can either 
```suggestion
    char s[] = "Hello World!";// ⤶
```
or simply 
```suggestion
    return 0;// ⤶
```

Reply:
Please do first